bf_downloads.html

```diff
@@ -361,18 +361,18 @@
                                 <td><a
                                         href="http://github.com/openmicroscopy/bioformats"
                                         target="_blank"><img
-                                        src="images/bf_gh_repo.png"
+                                        src="images/github_repository.png"
                                         alt="GitHub Repository Link" /></a></td>
                             </tr>
                             <tr>
                                 <td><a href="http://git.openmicroscopy.org/"
```

Comment:
Not this PR's issue, but I wonder if we want to link directly to http://git.openmicroscopy.org/?p=bioformats.git.